seshat/apps/core/models.py

```diff
@@ -1123,13 +1123,13 @@ def __str__(self) -> str:
 
 class VideoShapefile(models.Model):
     """
-    Model representing a video shapefile.
+    Model representing Cliopatria polity borders dataset.
     """
     id = models.AutoField(primary_key=True)
     geom = models.MultiPolygonField()
     simplified_geom = models.MultiPolygonField(null=True)
     name=models.CharField(max_length=100)
-    polity=models.CharField(max_length=100)
+    polity=models.CharField(max_length=100)  # Deprecated field (always same as name)
```

Comment:
These kind of deprecations would be good to take a look at soon, as they can bloat the database and slow things down.

Reply:
Yeah, I was just trying to avoid adding a db migration to this PR for the time being since it's a small change

Reply:
Ah, yes, good catch!!

Reply:
Actually perhaps I will remove it since I have added a migration anyway

Reply:
removed